### ISSUE TITLE
Use DAPR_REDIS_HOST for scaffolding instead of DAPR_NETWORK

### ISSUE THREAD

**Issue by philliphoff** [microsoft/vscode-dapr#84]
Currently the Dapr tasks scaffolder uses the environment variable `DAPR_NETWORK` to determine whether scaffold the Redis-related Dapr components file addresses using `localhost` (not present) or `redis` (present).  However, the latter may not match the actual Redis host name in that network (e.g. the Dev Container template uses `dapr_redis`).  Instead, the scaffolder should use `DAPR_REDIS_HOST` which is generally expected to be set when using a custom network.

**Comment by ElanHasson** [comment]
Hi @philliphoff,

I'm seeing an issue where the scaffolding is producing a redis container named `dapr_redis_dapr-dev-container`.

my `docker-compose.yml` looks like this:
```

      DAPR_NETWORK: dapr-dev-container
      DAPR_REDIS_HOST: dapr_redis
      DAPR_PLACEMENT_HOST: dapr_placement
```

**Comment by philliphoff** [comment]
@ElanHasson Yes, that's likely due to this bug (still present in `0.2.1` but fixed for a future `0.3.0`.  At some point I'd like to get our extension out of the business of scaffolding those files altogether; recent updates to the Dapr CLI should allow that.

**Comment by ElanHasson** [comment]
Cool thanks!
